transforms/dialog-full-screen-app-wrapper/__testfixtures__/Basic.input.js

```diff
@@ -0,0 +1,3 @@
+import DialogFullScreen from "carbon-react/lib/components/dialog-full-screen";
+
```

Comment:
I think this one should also have a test case for the content being defined as a variable, like: 

```js
  const content = (children) => (
    <div>
      { children }
    </div>
  );

  export const Test = ({ children }) => <DialogFullScreen>{ content(children) } </DialogFullScreen>
```

Reply:
Might well be the case that this already works with the codemod, in which case great but worth having the test case in IMO

Reply:
Yeah agree thanks